src/client/src/containers/About/index.tsx

```diff
@@ -38,6 +39,7 @@ const About = ({ versions, intl }: Props) => {
         <a
           className={styles.link}
           href="https://github.com/Microsoft/WebTemplateStudio"
+          onKeyUp={spaceKeyHandler}
```

Comment:
Would it make sense to name this function as `keyUpHandler` so that later we can use the same function to handle other key press events as well (if required) using the switch/case statement?

Reply:
yeah I didn't think about that but it would be better, changing that now